projects/packages/jitm/src/class-jitm.php

```diff
@@ -22,6 +22,19 @@ class JITM {
 
 	const PACKAGE_VERSION = '4.0.6';
 
+	/**
+	 * List of screen IDs where JITMs are allowed to display.
+	 *
+	 * @var string[]
+	 */
+	const APPROVED_SCREEN_IDS = array(
+		'jetpack',
+		'woo',
+		'shop',
+		'product',
+		'themes',
```

Comment:
Let's pull all these changes out into a separate PR, and remove the addition of `themes` to the list until we're reading to ship.

Reply:
See https://github.com/Automattic/jetpack/pull/41748

Reply:
```suggestion
```

No longer untangling Themes so let's remove.